docs/developer-guide/release.md

```diff
@@ -74,7 +75,8 @@ Note- the date should be the date of the release in the format sh run.sh make GH
       - click edit
       - add the name of the new release and the content from the change log text file above. 
       - commit to master
-1. Commit release files
+   3. Click on "Publish release" (THIS IS NEW, the release, so far should have been in Draft state)
+
    1. Commit all of the changed import and report files (ignore or discard diff files, change log) to a branch
```

Comment:
This is still unclear. This should be the order:

1. make all
2. commit PR
3. merge PR
4. make deploy
5. commit merge PR
6. Edit release

No more commits or anything after that.

Reply:
thanks for clarifying! I updated this, could you please let me know if this makes sense now? Thank you!